### DISCUSSION TITLE
Authentication is forced to my employer's single sign-on, can't deploy to my personal account

### DISCUSSION THREAD

**Discussion by gpadvorac** [community/community#28941] [Other Feature Feedback, Questions, & Ideas]
Two weeks ago I had to login to my employer's github account using single sign-on.  Now I'm in VS Code and trying to publish a project to my personal account but when VS Code opens a browser for authentication it always goes to the single sign-on.  I'm trapped.  I did a lot of searches and tried recommendations from here, but nothing helped.
https://stackoverflow.com/questions/15381198/remove-credentials-from-git/53605623#53605623

How can I unlock this so so I can choose which account to authenticate to.  In VS Code In even use the URL to the repo in my personal account, but it continues to do he single sign-on.

thanks.

**Comment by ettaboyle** [comment]
Hi @gpadvorac 👋🏻 There may be a workaround, but this sounds like an account-specific question. Your best bet will likely be opening a ticket on our [Support page](https://support.github.com/contact). 

Thanks for sharing the StackOverflow link too - Hopefully you can get this solved soon!

**Comment by benkovy** [comment]
I am also experiencing this, it is quite frustrating
